softgroup/services/authorization/authorization-impl/src/test/java/com/softgroup/authorization/AuthorizationCommandsIT.java

```diff
@@ -0,0 +1,53 @@
+package com.softgroup.authorization;
+
+import com.softgroup.authorization.api.message.LoginDataReq;
+import com.softgroup.authorization.api.message.LoginDataResp;
+import com.softgroup.authorization.impl.handler.AuthCmdLoginHandler;
+import com.softgroup.common.protocol.ActionHeader;
+import com.softgroup.common.protocol.Request;
+import com.softgroup.common.protocol.Response;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Spy;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class AuthorizationCommandsIT {
+
+    @Spy
+    AuthCmdLoginHandler cmdLoginHandler = new AuthCmdLoginHandler();
```

Comment:
@VasjaVn First of all you are not using verify. Next - you need to verify if some method was called. It's not applicable to data objects. Read about testing first.